en/middleware/micrortps.md

```diff
@@ -421,23 +421,16 @@ void sensor_combined_Subscriber::SubListener::onNewDataMessage(Subscriber* sub)
             std::cout << "\n\n\n\n\n\n\n\n\n\n";
             std::cout << "Sample received, count=" << n_msg << std::endl;
             std::cout << "=============================" << std::endl;
-            std::cout << "gyro_rad: " << st.gyro_rad().at(0);
-            std::cout << ", " << st.gyro_rad().at(1);
-            std::cout << ", " << st.gyro_rad().at(2) << std::endl;
-            std::cout << "gyro_integral_dt: " << st.gyro_integral_dt() << std::endl;
-            std::cout << "accelerometer_timestamp_relative: " << st.accelerometer_timestamp_relative() << std::endl;
-            std::cout << "accelerometer_m_s2: " << st.accelerometer_m_s2().at(0);
-            std::cout << ", " << st.accelerometer_m_s2().at(1);
-            std::cout << ", " << st.accelerometer_m_s2().at(2) << std::endl;
-            std::cout << "accelerometer_integral_dt: " << st.accelerometer_integral_dt() << std::endl;
-            std::cout << "magnetometer_timestamp_relative: " << st.magnetometer_timestamp_relative() << std::endl;
-            std::cout << "magnetometer_ga: " << st.magnetometer_ga().at(0);
-            std::cout << ", " << st.magnetometer_ga().at(1);
-            std::cout << ", " << st.magnetometer_ga().at(2) << std::endl;
-            std::cout << "baro_timestamp_relative: " << st.baro_timestamp_relative() << std::endl;
-            std::cout << "baro_alt_meter: " << st.baro_alt_meter() << std::endl;
-            std::cout << "baro_temp_celcius: " << st.baro_temp_celcius() << std::endl;
-
+            std::cout << "gyro_rad: " << st.gyro_rad_().at(0);
+            std::cout << ", " << st.gyro_rad_().at(1);
+            std::cout << ", " << st.gyro_rad_().at(2) << std::endl;
+            std::cout << "gyro_integral_dt: " << st.gyro_integral_dt_() << std::endl;
+            std::cout << "accelerometer_timestamp_relative: " << st.accelerometer_timestamp_relative_() << std::endl;
+            std::cout << "accelerometer_m_s2: " << st.accelerometer_m_s2_().at(0);
+            std::cout << ", " << st.accelerometer_m_s2_().at(1);
+            std::cout << ", " << st.accelerometer_m_s2_().at(2) << std::endl;
+            std::cout << "accelerometer_integral_dt: " << st.accelerometer_integral_dt_() << std::endl;
+            
```

Comment:
Please also add the `accelerometer_clipping`: https://github.com/PX4/Firmware/blob/master/msg/sensor_combined.msg#L20.